README.md

```diff
@@ -44,6 +44,9 @@ The files have the following meaning:
 In order to use this tool you need:
 * A Google account in the DIVD organisation
 * Create credentials.json using this page: https://developers.google.com/drive/api/v3/quickstart/python (make sure you are logged into you divd.nl account)
+    * Create a project and OAuth client ID using this page: https://console.cloud.google.com/projectselector2/apis/credentials
```

Comment:
Laten we hier wat voor bedenken. Ik kan de credentials.json ergens centraal klaar zetten.